README.md

```diff
@@ -79,46 +85,46 @@ curl -s -G --data-urlencode "q=title:list" localhost:9200/try_cloaked_search/_se
 These are a couple examples of simple term queries. They are still querying on the `title` field, which is unprotected.
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:Japan" localhost:8675/try_cloaked_search/_search | jq
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:cup" localhost:8675/try_cloaked_search/_search | jq
+./query-cloaked-search.sh '+tenant_id:"tenant-1" AND title:Japan'
+./query-cloaked-search.sh '+tenant_id:"tenant-1" AND title:cup'
 ```
 
 Compare these results to the ones returned by querying Elasticsearch directly. The same documents are returned, but the contents are much different.
 You can see how Cloaked Search transparently handles the decryption of the document to allow you to see the data in the fields that were protected, `summary` and `body`.
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:Japan" localhost:9200/try_cloaked_search/_search | jq
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:cup" localhost:9200/try_cloaked_search/_search | jq
+./query-search-service.sh '+tenant_id:"tenant-1" AND title:Japan'
+./query-search-service.sh '+tenant_id:"tenant-1" AND title:cup'
 ```
 
 Now try querying on a protected field:
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND summary:glasgow" localhost:8675/try_cloaked_search/_search | jq
+./query-cloaked-search.sh '+tenant_id:"tenant-1" AND summary:glasgow'
 ```
 
 Term queries can also be combined with ORs or ANDs, and you can mix protected and unprotected fields. For example,
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND (title:cup OR title:Japan)" localhost:8675/try_cloaked_search/_search | jq
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND (summary:cup OR title:Japan)" localhost:8675/try_cloaked_search/_search | jq
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND (summary:cup OR body:Japan)" localhost:8675/try_cloaked_search/_search | jq
+./query-cloaked-search.sh '+tenant_id:"tenant-1" AND (title:cup OR title:Japan)'
+./query-cloaked-search.sh '+tenant_id:"tenant-1" AND (summary:cup OR title:Japan)'
+./query-cloaked-search.sh '+tenant_id:"tenant-1" AND (summary:cup OR body:Japan)'
 ```
 
 Phrases can also be searched using quoted queries like this:
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND summary:\"Cheerleading in Japan\"" localhost:8675/try_cloaked_search/_search | jq
+./query-cloaked-search.sh '+tenant_id:"tenant-1" AND summary:"Cheerleading in Japan"'
 ```
 
 Finally, here is an example of a prefix query:
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND summary:list*" localhost:8675/try_cloaked_search/_search | jq
+./query-cloaked-search.sh '+tenant_id:"tenant-1" AND summary:list*'
```

Comment:
I know this is a query from before, but it's not a very good prefix query as all the results are just for "List", so the substring isn't adding any visible benefit